openam-jp/openam
# Issue #187 Document URL

## Analysis

After fix of #135, the layout and images of OpenAM have been changed to those unique to the OpenAM consortium.

However, the FoorgeRock URLs remain in the following places.

* Product Documentation link in Common Tasks
* Release note link in Upgrade screen

## Solution

Change the URLs to OpenAM consortium GitHub Wiki page (https://github.com/openam-jp/openam/wiki).

The following files are not targeted for modification, since it is unclear whether they are being used.

* openam-core/src/main/java/org/forgerock/openam/console/ui/taglib/propertysheet/CCPropertySheetTag.java
* openam-console/src/main/webapp/console/task/Home.jsp

## Testing

* Check `Product Documentation` link in Common Tasks
* Check upgrade screen (English and Japanese)
